packages/example/src/pages/components/ExpressiveList.mdx

```diff
@@ -0,0 +1,135 @@
+---
+title: ExpressiveList
+description: Usage instructions for the FeatureCard component
+---
+
+import { Connect } from '@carbon/pictograms-react';
+
+<PageDescription>
+
+The `<ExpressiveList>` component is meant to be used with the `expressive-content-list-group` className when grouped together. This component is only intended for paragraph content and not meant to nest other components within it. `ExpressiveListContainer` coming soon. This can be used with a pictogram by importing a pictogram from our library and passing it into the `picogram` prop as a node.
```

Comment:
```suggestion
The `<ExpressiveList>` component is meant to be used with the `expressive-content-list-group` className when grouped together. This component is only intended for paragraph content and not meant to nest other components within it. `ExpressiveListContainer` coming soon. This can be used with a pictogram by importing a pictogram from our library and passing it into the `pictogram` prop as a node.
```